src/components/ImageDetails/PhotoDetails.vue

```diff
@@ -31,8 +31,8 @@
                            :providerName="providerName" />
     </div>
     </div>
-    <div class="column image-info">
-      <div class="margin-vertical-normal">
+    <div class="column image-info" id="right-side">
```

Comment:
Please use the Vocabulary margin classes https://cc-vocabulary.netlify.app/?path=/docs/tokens-spacing--figma#usage

Reply:
@brenoferreira Thank you for reviewing my pr. I have updated the suggested changes, thanks.